### DISCUSSION TITLE
How to support x64 in bundle of wix3

### DISCUSSION THREAD

**Discussion by chancezheng** [wixtoolset/issues#8661] [Questions]
when i setting the property 'platform=x64', but the package of bootstrapper is x86 architecture

**Comment by bevanweiss** [comment]
WiX v3 does not support 64 bit bundles.
You will need to migrate to the latest supported version of WiX if you need 64 bit bundles / bootstrapping.

https://www.joyofsetup.com/2021/06/05/wix-v4-random-fact-no.-4/

**Reply by chancezheng** [reply]
ok, 3q
